pyproject.toml

```
@@ -94,6 +89,35 @@ select = [
     # "PYI061", # TODO: Enable when out of preview
     "PYI062", # Duplicate literal member `{}`
     "PYI064", # `Final[Literal[{literal}]]` can be replaced with a bare Final
+    # flake8-simplify, excluding rules that can reduce performance or readability due to long line formatting
+    "SIM101", # Multiple `isinstance` calls for `{name}`, merge into a single call
+    "SIM103", # Return the condition `{condition}` directly
+    "SIM107", # Don't use return in `try-except` and `finally`
+    "SIM109", # Use `{replacement}` instead of multiple equality comparisons
+    "SIM112", # Use capitalized environment variable `{expected}` instead of `{actual}`
+    "SIM113", # Use `enumerate()` for index variable `{index}` in `for` loop
+    "SIM114", # Combine `if` branches using logical `or` operator
+    "SIM115", # Use a context manager for opening files
+    "SIM118", # Use key `{operator}` dict instead of key `{operator} dict.keys()`
+    "SIM201", # Use `{left} != {right}` instead of not `{left} == {right}`
+    "SIM202", # Use `{left} == {right}` instead of not `{left} != {right}`
+    "SIM208", # Use `{expr}` instead of `not (not {expr})`
+    "SIM210", # Remove unnecessary `True if ... else False`
+    "SIM211", # Use `not ...` instead of `False if ... else True`
+    "SIM212", # Use `{expr_else} if {expr_else} else {expr_body}` instead of `{expr_body} if not {expr_else} else {expr_else}`
+    "SIM220", # Use `False` instead of `{name} and not {name}`
+    "SIM221", # Use `True` instead of `{name} or not {name}`
+    "SIM222", # Use `{expr}` instead of `{replaced}`
+    "SIM223", # Use `{expr}` instead of `{replaced}`
```

Could be
```suggestion
    "SIM2", # flake8-simplify conditional ordering rules
```
but as asked I explicitly selected them individually